src/vcpkg/tools.cpp

```diff
@@ -567,6 +567,35 @@ gsutil version: 4.58
         }
     };
 
+    struct AWSCliProvider : ToolProvider
```

Comment:
```suggestion
    struct AwsCliProvider : ToolProvider
```

Acronyms should be treated as a single word for the purposes of camel case, like in [rust].

[rust]: https://rust-lang.github.io/api-guidelines/naming.html